projects/cron-editor/src/lib/CronOptions.ts

```diff
@@ -14,5 +14,8 @@ export interface CronOptions {
     hideYearlyTab: boolean;
     hideAdvancedTab: boolean;
     use24HourTime: boolean;
-    hideSeconds: boolean;
+    hideSeconds: boolean; //hides the seconds field in the UI
+
+    removeSeconds: false; // removes seconds from the CRON expression
```

Comment:
Please make these two members `boolean`, otherwise you can only assign false to them